File: src/main/java/com/dh/ondot/member/core/exception/TokenBlacklistedException.java
package com.dh.ondot.member.core.exception;

import com.dh.ondot.core.exception.UnauthorizedException;

import static com.dh.ondot.core.exception.ErrorCode.TOKEN_BLACKLISTED;

public class TokenBlacklistedException extends UnauthorizedException {

  public TokenBlacklistedException() {
    super(TOKEN_BLACKLISTED.getMessage());
  }

  @Override
  public String getErrorCode() {
    return TOKEN_BLACKLISTED.name();
  }
}

File: src/main/kotlin/com/dh/ondot/member/core/exception/TokenBlacklistedException.kt
package com.dh.ondot.member.core.exception

import com.dh.ondot.core.exception.ErrorCode.TOKEN_BLACKLISTED
import com.dh.ondot.core.exception.UnauthorizedException

class TokenBlacklistedException :
    UnauthorizedException(TOKEN_BLACKLISTED.message) {
    override val errorCode: String get() = TOKEN_BLACKLISTED.name
}
